tests/api/iotrace_parser.py

```diff
@@ -0,0 +1,173 @@
+#
```

Comment:
I would name the parser something like iotrace_stats_parser.py
Unless we will put put parsing of every iotrace command here later on